### PR TITLE
Updates for the new setup

### PR DESCRIPTION
**Pull request by mohierf**
Remove dependencies with alignak_setup

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -3,34 +3,29 @@ Alignak Example Module
...
-.. image:: https://coveralls.io/repos/Alignak-monitoring-contrib/alignak-module-example/badge.svg?branch=develop&service=github
-    :target: https://coveralls.io/github/Alignak-monitoring-contrib/alignak-module-example?branch=master
-    :alt: Code coverage
+.. image:: https://coveralls.io/repos/Alignak-monitoring-contrib/alignak-module-example/badge.svg?branch=develop
+    :target: https://coveralls.io/r/Alignak-monitoring-contrib/alignak-module-example
+    :alt: Development code tests coverage
 
-Most recent release
--------------------
+.. image:: https://badge.fury.io/py/alignak_module_backend.svg
```
Comment (ddurieux) [inline README.rst]: error, not right link

File: README.rst [hunk 2]
```diff
@@ -44,30 +39,24 @@ Packaging
...
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Modules can be attached to an Alignak daemon thanks to the daemon configuration file. The module
-defines, in its own properties which daemon it may be attached to and the module documentation
-will informe the user about this.
+Modules can be attached to an Alignak daemon thanks to the daemon configuration file. The module defines, in its own properties which daemon it may be attached to and the module documentation will informe the user about this.
 
-Rather than Shinken, a module cannot have sub-modules. This feature is not currenlty well tested
-and robust :)
+Rather than Shinken, a module cannot have sub-modules. This feature is not currenlty well tested and robust enough :)
```
Comment (ddurieux) [inline README.rst]: typo `currently`

File: README.rst [hunk 3]
```diff
@@ -87,15 +76,14 @@ Repository directories and files example::
...
+        etc/
             arbiter/
                 modules/
                     mod-EXAMPLE.cfg
         __init__.py
         EXAMPLE.py
 
-The content of the directory ``alignak_checks_EXAMPLE/ALIGNAKETC`` (including files and sub
-directories) will be copied to */usr/local/var/etc/alignak*.
+The content of the directory ``alignak_checks_EXAMPLE/ALIGNAKETC`` (including files and sub directories) will be copied to */usr/local/var/etc/alignak*.
```
Comment (ddurieux) [inline README.rst]: modify because ALIGNAKETC no longer exist in with this modification

**Review by ddurieux — CHANGES_REQUESTED** [review]

File: README.rst (hunk 1 above)
Comment (mohierf) [reply README.rst]: ok

**Review by ddurieux — APPROVED** [review]

**Pull request merged**